src/cli/commands/force/lightning/local/__tests__/setup.test.ts

```diff
@@ -1,38 +0,0 @@
-/*
```

Comment:
We should keep Setup and its tests in this repo. It's a pass-through command so that if a user installs `lwc-dev-mobile` they could run the Setup command without having to install `lwc-dev-mobile-core`. Of course the other command won't be extending from Setup anymore but we still want to keep it for this reason.